### PR TITLE
7374 report add missing commands

### PR DESCRIPTION
**Pull request by coherit**
Adds functionality for update, unregister, and list report templates

-   [X] Bugfix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

The report management command had handles for four operations: register, update, unregister, and list, but only the register function had been written. This PR adds the missing update, unregister, and list functions, following the same pattern of similar management commands (e.g. widget).

### Issues Solved

#7374 

### Checklist

-   [X ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: Self funded
*   Found by: @bferguso
*   Tested by: @coherit
*   Designed by: @coherit

### CODE REVIEW

File: arches/__init__.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 except ModuleNotFoundError as e:
     print(e)
 
-VERSION = (6, 0, 0, "beta", 0)  # VERSION[3] options = "alpha", "beta", "rc", or "final"
+VERSION = (6, 0, 0, "final", 0)  # VERSION[3] options = "alpha", "beta", "rc", or "final"
```
Comment (chiatt) [inline arches/__init__.py:9]: Could you change the version back to "beta"?

File: arches/management/commands/report.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 
 import os
 import uuid
+import imp
```
Comment (chiatt) [inline arches/management/commands/report.py:21]: I don't see the `imp` package used. Could you remove that import statement?

**Review by chiatt — CHANGES_REQUESTED** [review]
Thanks @coherit for adding in these commands. This looks really good. Just a couple of really minor changes to make and this will be ready to merge.

File: arches/__init__.py (hunk 1 above)
Comment (coherit) [reply arches/__init__.py]: done

File: arches/management/commands/report.py (hunk 2 above)
Comment (coherit) [reply arches/management/commands/report.py]: yep - done

**Review by chiatt — APPROVED** [review]
Looks good. Thanks @coherit!

**Pull request merged**